container-images/cuda-vllm/Containerfile

```diff
@@ -15,3 +12,24 @@ WORKDIR /src/ramalama
 RUN container-images/scripts/build-vllm.sh "cuda"
 WORKDIR /
 
+# Final runtime image
+FROM docker.io/nvidia/cuda:${CUDA_VERSION}-runtime-ubi9
+
+COPY --from=builder /opt/venv /opt/venv
+COPY --from=quay.io/ramalama/cuda:latest /usr /usr
+
+# Workaround for CUDA libraries not in the ld path in base container
+RUN echo "/usr/local/cuda-12.8/compat" > /etc/ld.so.conf.d/99_cuda_compat.conf && ldconfig
+
+# Copy the entire installation directory from the builder
```

Comment:
Are these lines necessary or should they be deleted?

Reply:
Not sure yet, still trying to build this stuff, the iterations are long because this step takes ages:

MAX_JOBS=2 python3 setup.py install

Then I guess I need to test and see. I don't own Nvidia hardware though, so I'm hoping it will just fallback to CPU, so I can do some basic testing.

I might need some help from someone with a working Nvidia system to test

Reply:
I'm actually gonna change this a bit dunno what I was thinking, the runtime images should be:

FROM quay.io/ramalama/cuda:latest